core/modules/taxonomy/taxonomy.module

```diff
@@ -388,13 +388,11 @@ function taxonomy_admin_paths() {
  * @see taxonomy_menu()
  */
 function taxonomy_term_access($op, $term) {
-  if (!$term || !in_array($op, array('edit', 'delete'), TRUE)) {
-    // If there was no term to check against, or the $op was not one of the
-    // supported ones, we return access denied.
-    return FALSE;
+  if ($op == 'edit') {
+    // Normalize edit op to update.
+    $op = 'update';
```

Comment:
Nice, thanks for adding this BC here. Similar to comment "edit" calls, filing a `WATCHDOG_NOTICE` here (if the site is so configured) would be good to help modules upgrading going further.

As this is an API change, adding a `@since` line here is needed as well.

Reply:
Fixed in https://github.com/backdrop/backdrop/pull/2451